samples/highcharts/demo/calendar-heatmap-month/demo.css

```diff
@@ -0,0 +1,17 @@
+.highcharts-figure {
+    min-width: 375px;
+    max-width: 800px;
+    margin: 1em auto;
+}
+
+.date {
+    text-align: center;
+    color: rgba(70, 70, 92, 1);
+    margin: 0;
+    padding: 2px 2px 0;
+    min-width: 14px;
+    font-size: 0.8rem;
+    font-weight: bold;
+    background-color: whitesmoke;
+    opacity: 0.5;
+}
```

Comment:
It appears like a hack with this custom class and CSS rule in a chart that is otherwise pure JS configuration. As far as I can see, all of this can be achieved by configuring data label. Turn off `useHTML` and reduce the paragraph inside the `format`. Use `backgroundColor` and `padding` to achieve the box effect. Set `style.textOutline` to `none`.